compiler/optimizer/OMROptimizer.cpp

```diff
@@ -1566,7 +1566,6 @@ int32_t OMR::Optimizer::performOptimization(const OptimizationStrategy *optimiza
             bool blocksArePending = false;
             while (subGroup->_num != endGroup && subGroup->_num != endOpts)
                {
-               OMR::Optimizations optNum = subGroup->_num;
```

Comment:
I'm not sure this is right the definition of optNum way back in the file is for the optimization overall and not the number within the subgroup - @vijaysun-omr do you agree?